lighthouse-cli/test/smokehouse/error-expectations.js

```diff
@@ -18,10 +18,10 @@ module.exports = [
     },
   },
   {
-    errorCode: 'INSECURE_DOCUMENT_REQUEST',
+    errorCode: undefined,
```

Comment:
this should have other expectations then? And maybe a comment that this is ensuring there *isn't* an error (since it's in the `error-expectations` file :)

Reply:
This should be fixed by followup to #8340, the comments I have there address the fact that none of these are currently fatal errors but IMO should be non-zero exit codes.